src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/google/HttpGoogleDataprocDAO.scala

```diff
@@ -44,6 +45,7 @@ class HttpGoogleDataprocDAO(appName: String,
 
   private lazy val oauth2Scopes = List(Oauth2Scopes.USERINFO_EMAIL, Oauth2Scopes.USERINFO_PROFILE)
   private lazy val bigqueryScopes = List(BigqueryScopes.BIGQUERY)
+  private lazy val cloudSourceRepositoryScopes = List(CloudSourceRepositoriesScopes.SOURCE_FULL_CONTROL)
```

Comment:
Scope descriptions: https://developers.google.com/identity/protocols/googlescopes#sourcerepov1

Technically I think Nicole just wants _read_ scopes, but I don't see any harm in granting full control to the project's source repos.

Reply:
Nvm, checked with Brad, going to make this read scopes.